README.md

```diff
@@ -1 +1,169 @@
-# capstone-project-1
\ No newline at end of file
+# startup-society-summit
+It's Microverse program capstone (module-1) project
+
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Startup society summit</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```

Comment:
- [ ] Seems you forgot this section of your README. Kindly add 1 to 3 future features that you might possibly add to the project. This was placed with best practices in mind for good documnetion of your project 👍